homeassistant/components/octoprint/__init__.py

```diff
@@ -51,10 +54,32 @@ def ensure_valid_path(value):
     return value
 
 
+class OctoprintBinarySensorMetadata(NamedTuple):
+    """Metadata for an individual Skybell binary_sensor."""
+
+    name: str
+    endpoint: str
+    group: str
+    key: str
+    unit: str | None
```

Comment:
Let's call it `unit_of_measurement` to be consistent

Reply:
Done